addon/templates/components/polaris-resource-list/item.hbs

```diff
@@ -69,7 +73,7 @@
     </div>
   {{/if}}
 
-  {{#if shortcutActions}}
+  {{#if (and shortcutActions (not loading))}}
     {{#wrapper-element
```

Comment:
why do we use this `wrapper-element` here?

Reply:
I think to handle the `click` and gain access to the event so that we can [stop it propagating](https://github.com/smile-io/ember-polaris/blob/3861a68eef235a99a42346bafdb320dc6eab44aa/addon/templates/components/polaris-resource-list/item.hbs#L80) I think, this was from a while ago 😅

Reply:
kk